app/controllers/application_controller.rb

```diff
@@ -12,9 +12,11 @@ class ApplicationController < ActionController::Base
   def current_profile = current_user&.profile
 
   # TODO: Must change after implementing multi-conference support
-  def current_conference = Conference.last
+  def current_conference
+    @_current_conference ||= Conference.last
+  end
 
   def vapid_public_key
-    Base64.urlsafe_decode64(ENV["VAPID_PUBLIC_KEY"]).bytes.to_json
+    @_vapid_public_key ||= Base64.urlsafe_decode64(ENV["VAPID_PUBLIC_KEY"]).bytes.to_json
```

Comment:
This one isn't a big deal but since it's[ computed every time time main layout renders](https://github.com/TelosLabs/rails-world/blob/9c20cbb35fd1b81162b98953f7da212730cbfefe/app/views/layouts/application.html.erb#L34) it seemed worth memoising as well.